app/controllers/bus_stops_controller.rb

```diff
@@ -61,13 +61,12 @@ def name_search
 
   def outdated
     @date = Date.parse(params[:date]) rescue 1.month.ago.to_date
-    @stops = BusStop.not_updated_since(@date)
-                    .order(:updated_at)
-                    .paginate(page: params[:page], per_page: 10)
+    outdated_stops = BusStop.not_updated_since(@date).order(:updated_at)
+    @stops = outdated_stops.paginate(page: params[:page], per_page: 10)
```

Comment:
I would argue that the more intuitive way to break this down, since it's the request format that makes the difference between whether we paginate or not, would be to put the pagination code inside only the `format.html` block.

Having an instance variable that only gets *used* in that context, but is defined for both, seems like not quite clearly expressing our intent.

Reply:
Ah ok, so you're saying to remove one variable completely and then paginate on that in format.html

Reply:
Yes, if you agree that it would be more intuitive that way.

Reply:
Same thing with `manage`?

Reply:
Yes, I think in general when the choice of response format is what means you have different data, it's good to structure the branching to reflect that.